apis/python/examples/soma-slice-query.md

```diff
@@ -0,0 +1,200 @@
+Here we show an example of doing a _slice query_ across a `SOMACollection`.
+
+## Populate the collection
+
+Here we use a few small sample files included in this repository.
+
+```
+import tiledbsc
+import tiledbsc.io
+import os
+import shutil
+
+soco_path = './soco-attribute-filter'
+if os.path.exists(soco_path):
+    shutil.rmtree(soco_path)
+
+soco = tiledbsc.SOMACollection(soco_path)
+if not soco.exists():
+    soco._create()
```

Comment:
I would drop the underscore here, as this is not a private method (since it's being used externally). Also, a `create_if_not_exists()` method could be helpful.